setup.md

```diff
@@ -230,7 +232,7 @@ $ git config --list
 {: .bash}
 
 ## Create GitHub Account <a name="github_account"></a>
-Create an account on [github.com] if you do not have one already. Remember the user name and password. If you are making a GitHub account, please remember that your username should be *recognizable* and *professional*.
+Create an account on [github.com](https://github.com/) if you do not have one already. Remember the username and password. If you are making a GitHub account, please remember that your username should be *recognizable* and *professional*.
```

Comment:
This link was originally generated from a link at the bottom of the page. Please correct the url at this location too to avoid future problems.